src/map/utils/battleutils.cpp

```diff
@@ -730,6 +730,16 @@ namespace battleutils
 
                         if (PDefender->isAlive())
                         {
+                            // Subpower is the remaining damage that can be drained. When it reaches 0 the effect ends
+                            int remainingDrain = PDefender->StatusEffectContainer->GetStatusEffect(EFFECT_DREAD_SPIKES)->GetSubPower();

```

Comment:
The request already confuses me. We'd either be calling just dread spikes PStatus when none of the others are written that way, or we'd need to redefine PStatus for each one. This is more confusing, not less.